### PR TITLE
Improve code quality

### PR DESCRIPTION
**Pull request by McNaBry**

### CODE REVIEW

**Comment by CJ-Lee01** [comment]
Is this ready for review?

**Comment by McNaBry** [comment]
> Is this ready for review

Yes

File: src/main/java/seedu/address/logic/commands/AddRecruiterCommand.java
Comment (CJ-Lee01) [inline src/main/java/seedu/address/logic/commands/AddRecruiterCommand.java:1]: Good refactor

**Review by CJ-Lee01 — APPROVED** [review]
Looks good to me.

**Comment by CJ-Lee01** [comment]
Looks like there is no need to update the branch.

**Pull request merged**